cartesian_controller_handles/src/motion_control_handle.cpp

```diff
@@ -103,6 +103,7 @@ controller_interface::InterfaceConfiguration
 MotionControlHandle::command_interface_configuration() const
 {
   controller_interface::InterfaceConfiguration conf;
+  conf.type = controller_interface::interface_configuration_type::INDIVIDUAL;
```

Comment:
```suggestion
  conf.type = controller_interface::interface_configuration_type::NONE;
```

That's in my opinion even clearer. The handles should not claim command handles.

Reply:
I just checked this and it works as well :+1: